frontend/BossAlienMediaPlayer/src/app/http.service.ts

```diff
@@ -43,6 +43,10 @@ export class HttpService {
       err => 
       {
         this.requestInFlight = false;
+        if (err.status == 404) // This prevents the pesky error from /api/queue when not logged in
```

Comment:
Hmm, I'm not sure blanket-ignoring all 404's is the best solution to this problem. Maybe better would be to only send the requests if we know we are logged in?

Reply:
Ok, I have a better understanding on how to develop with Angular after doing the listeners thing, I'll double check the queue component, which I think is the culprit for this

Reply:
Still, retrying a 404 makes no sense, so I think it should stay

Reply:
I've located where the extra call was done, I'll still keep this one for the above reason